### DISCUSSION TITLE
Vote to retire @ryanmoran

### DISCUSSION THREAD

**Discussion by dmikusa** [paketo-buildpacks/feedback#272] [Utilities Team]
Hey https://github.com/orgs/paketo-buildpacks/teams/utilities-maintainers, I'd like to propose to move @ryanmoran to an emeritus maintainer position. I do not believe they are involved in the project anymore. As is traditional, we will award the emeritus status as recognition for helping with the project.

@paketo-buildpacks/utilities-maintainers Can I get a 👍 / 👎 vote?

**Comment by ForestEckhardt** [comment]
There appears to be a super majority. @ryanmoran  will be awarded with emeritus status.
